src/components/ThirdPartyIntegration.js

```diff
@@ -22,21 +30,74 @@ export const ThirdPartyIntegration = ({
   } = useSWR(id, IntegrationsSdk.status);
 
   const [isConnectionActive, setConnectionActive] = useState(false);
+  const [openSection, setOpenSection] = useState(true);
+  useLayoutEffect(() => {
+    const urlParams = new URLSearchParams(window.location.search);
+    const currentSection = urlParams.get('section');
+
+    if (typeof integrationStatus !== 'undefined') {
+      if (openSection && currentSection === 'paypal') {
+        const ppButton = document.querySelector('.yith-btn-paypal');
+
+        if (ppButton?.dataset?.securewindowmsg && document.getElementById('signup-js') !== 'undefined') {
+          ppButton.click();
+          ppButton.disabled = true;
+          setOpenSection(false);
+        }
+      }
+
+      if (openSection && currentSection === 'shippo') {
+        const shippoButton = document.querySelector('.section-shippo button');
+        const event = new Event('onboarding-shippo-popup');
+        window.document.dispatchEvent(event);
+        setOpenSection(false);
+      }
+    }
+  });
+
 
   let installPlugin = useSWRMutation("install-plugin", async () => {
     await PluginsSdk.actions.installSync(
       integrationStatus?.integration?.plugin?.slug
     );
     await refreshIntegrationStatus();
     setConnectionActive(true);
+    if (id === 'shippo' || id === 'paypal') {
+      window.location.reload();
+      if(  window.location.search.indexOf('section') !== -1 ){
+        window.location.search = replaceQueryParam('section', id,  window.location.search);
+      }else{
+        window.location.search += ('&section=' + id);
+      }
+
+    }
   });
+  const updateButtonEvent = async (event) => {
+    if (event.detail.connected == 1) {
+      const integrationStatusResponse =
+          await refreshIntegrationStatus();
+      if (integrationStatusResponse.complete) {
+        notify.push(`${id}-account-connect-success`, {
+          title: `Your ${id} account have been connected`,
```

Comment:
@sumathi3399 Can we plz add translations texts here?

Reply:
done